app/containers/HomePage/About.js

```diff
@@ -0,0 +1,357 @@
+        padding-bottom: 32px;
+        letter-spacing: -1.2px;
+      }
+
+      .text-3 {
+        font-family: OpenSans;
+        color: #282828;
+        font-size: 20px;
+        letter-spacing: -0.8px;
+      }
+    }
+  }
+
+  @media only screen and (max-width: 992px) {
+    .second-screen-header {
+      padding-top: 81px;
+      padding-bottom: 0;
+    }
+
+    .main-difference {
+      li {
+        padding-top: 75px;
+        padding-bottom: 0;
+      }
+    }
+
+    .second-screen-elements-column {
+      li {
+        padding-top: 40px;
+
+        .block1 {
+          margin: 40px 0;
+          text-align: center;
+        }
+
+        .block1,
+        .block2 {
+          order: 1 !important;
+        }
+      }
+    }
+  }
+`;
+
+const About = () => (
+  <Box id={SECOND_SCREEN}>
+    <div className="container second-screen">
+      <h1 className="col-lg-12 second-screen-header">
+        <FormattedMessage {...messages.differentThan} />
+      </h1>
+      <div className="row second-screen-elements">
+        <ul className="col-lg-12 main-difference">
+          <div className="row">
+            <li className="col-lg-4">
+              <div className="card row">
+                <div className="col-lg-12">
+                  <img src={img1} alt="img1" />
+                </div>
+                <div className="card-body">
+                  <p className="card-text item-title">
+                    <FormattedMessage {...messages.practicalAnswers} />
+                  </p>
+                  <p className="card-text item-content">
+                    <FormattedMessage {...messages.sitesAreModerated} />
+                  </p>
+                </div>
+              </div>
+            </li>
+            <li className="col-lg-4">
+              <div className="card row">
+                <div className="col-lg-12">
+                  <img src={img2} alt="img2" />
+                </div>
+                <div className="card-body">
+                  <p className="item-title">
+                    <FormattedMessage {...messages.fairEconomy} />
+                  </p>
+                  <p className="item-content">
+                    <FormattedMessage {...messages.postsOnQA} />
+                  </p>
+                </div>
+              </div>
+            </li>
+            <li className="col-lg-4">
+              <div className="card row">
+                <div>
+                  <img src={img3} alt="img3" />
+                </div>
+                <div className="card-body">
+                  <p className="item-title">
+                    <FormattedMessage {...messages.builtOnSmart} />
+                  </p>
+                  <p className="item-content">
+                    <FormattedMessage {...messages.platformIsDistributed} />
+                  </p>
+                </div>
+              </div>
+            </li>
+          </div>
+        </ul>
+      </div>
+    </div>
+    <div className="separator" />
```

Comment:
Use padding on the section instead of adding separator. You can use selectors `:first-xxx`, `:last-xxx` is the style applies only to this section.